### PR TITLE
Add Solar-Log integration docs

### PR DESCRIPTION
**Pull request by Ernst79**
**Description:**
Solar-Log documentation for the new Solar-Log platform

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27036

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/solarlog.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: "Solar-Log Sensor"
+description: "Instructions on how to integrate Solar-Log sensors within Home Assistant."
+logo: solar-log.png
+ha_category: Sensor
+ha_release: 0.100
```
Comment (klaasnicolaas) [inline source/_components/solarlog.markdown]:
```suggestion
ha_release: "0.100"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
What is the purpose of the zone.identifier files? I think you can leave it out.

File: source/_components/solarlog.markdown (hunk 1 above)
Comment (Ernst79) [reply source/_components/solarlog.markdown]: Fixed, I also added double quotes around the IoT class.

I've removed the zone.identifier files. I don't know why, but they were created automatically.

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

Happy Hacktoberfest! 🎉

**Comment by Ernst79** [comment]
@klaasnicolaas Balloob requested some improvements to the integration, which also had to be added to the docs. Can you review the changes?
- Added the option to use configuration in the user interface.
- Removed the requirement to not have http:// in the host address (will be automatically added now if not provided).

**Pull request merged**